punishments.js

```diff
@@ -537,6 +577,7 @@ Punishments.roomPunish = function (room, user, punishment, recursionKeys) {
 	if (user.trusted) {
 		Punishments.roomUserids.nestedSet(room.id, user.trusted, punishment);
 		keys.add(user.trusted);
+		// @ts-ignore
```

Comment:
You know, TypeScript has a point here. Is this a bug?

Reply:
This entire function is kind of weird... Oh well, I'll fix it later I guess.

Reply:
Yeah I wasn't entirely sure what was going on here, since `affected` later on is only used if it is defined so I was confused why it was being used here, I'll add a TODO to look at this I guess.